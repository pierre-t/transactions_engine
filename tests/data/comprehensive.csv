type,client,tx,amount
deposit,1,1,1000.0
deposit,2,2,2000.0
deposit,1,3,500.0
withdrawal,1,4,200.0
withdrawal,2,5,1000.0
dispute,1,1,
withdrawal,1,6,100.0
resolve,1,1,
deposit,1,7,300.0
dispute,1,3,
chargeback,1,3,
deposit,4,101,1000.0
deposit,42,8,1000.0
withdrawal,42,9,999.9999
deposit,3,10,100.0
deposit,3,11,200.0
dispute,3,10,
withdrawal,4,102,1000.0
