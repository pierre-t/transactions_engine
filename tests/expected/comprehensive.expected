client,available,held,total,locked
1,1000,0,1000,true
2,1000,0,1000,false
3,200,100,300,false
4,0,0,0,false
42,0.0001,0,0.0001,false
43,499.5000,0,499.5000,false
44,499.4999,0,499.4999,false
